Index: testProtos.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- testProtos.sh	(revision a7b71a82fc0c1174de5879f75932df32ebe8a6df)
+++ testProtos.sh	(revision bba5a9267330d78eb0fe7a8a9c92d22e2256e1a4)
@@ -299,12 +299,14 @@
 TestProto     SingleModelProto                            mainApp
 TestProto     SingleModelOpenMPProto                      mainApp
 
+echo "== TEST SUMMARY START =="
 i=1
 while [[ $i -le $count ]]
 do
 echo ${testResult[i]}: ${testList[i]}
 ((i++))
 done
+echo "== TEST SUMMARY STOP =="
 
 echo
 echo ---------------------------------------------------------------------------
